strawberry_django/mutations/resolvers.py

```diff
@@ -67,67 +66,71 @@ def _parse_pk(
         return value.parse(model)
 
     if isinstance(value, dict):
+        if key_attr in value:
+            obj_pk = value[key_attr]
+            if obj_pk != strawberry.UNSET:
+                if key_attr == "id":
+                    obj_pk = int(obj_pk)
+                return model._default_manager.get(pk=obj_pk), value
         return None, value
 
     return model._default_manager.get(pk=value), None
 
 
-def _parse_data(info: Info, model: type[_M], value: Any):
-    obj, data = _parse_pk(value, model)
-
+def _parse_data(info: Info, model: type[_M], value: Any, key_attr: str):
```

Comment:
Ditto, and the same applies to every other places in this PR

```suggestion
def _parse_data(info: Info, model: type[_M], value: Any, *, key_attr: str = "id"):
```

Reply:
Thank you! I am not sure about the default value, shouldn't it be `pk`, as this is the default value set in the `DjangoCUDMutation` (https://github.com/strawberry-graphql/strawberry-graphql-django/blob/main/strawberry_django/mutations/fields.py#L178)

Reply:
A friendly ping @bellini666  :)

Reply:
@tokr-bit oh, sorry, somehow I lost the previous notification :)

Yeah I think you are correct! The default is usually `pk` because it will work for any model's primary key even if you set it to something other than `id`

The `id` itself needs to be checked for relay stuff

Reply:
@tokr-bit Thanks for getting back to me! I followed your requests. In my opinion, the branch is ready to be merged.